src/package-resolver.js

```diff
@@ -388,14 +388,18 @@ export default class PackageResolver {
       }
     }
 
+    if (getExoticResolver(version) && manifest) {
+      this.exoticRangeMatch(patterns.map(p => this.getStrictResolvedPattern(p)), manifest);
```

Comment:
You can do `patterns.map(this.getStrictResolvedPattern.bind(this)), manifest);` instead.